tardis/adapters/sites/moab.py

```diff
@@ -210,6 +212,15 @@ async def stop_resource(self, resource_attributes: AttributeDict):
         logger.debug("MOAB jobs cannot be stopped gracefully. Terminating instead.")
         return await self.terminate_resource(resource_attributes)
 
+    def msub_cmdline_options(self):
+        cmd_string = scheduler_cmd_option_formatter(
+            self.machine_type_configuration.get("SubmitOptions", AttributeDict())
+        )
+        # Add trailing space at end if not already present
+        if cmd_string != "" and not cmd_string.endswith(" "):
```

Comment:
If you follow my suggestion to include all cmd line options in the `AttributeDicts` passed to `submit_cmdline_options` call from above, you can remove that lines.